playbooks/roles/handle-nodes-on-openstack/tasks/create_on_openstack.yml

```diff
@@ -37,6 +37,7 @@
         hosts:
           {{ servername }}:
             ansible_host: {{ floating_ip }}
+            ansible_python_interpreter: {{ ansible_python_interpreter }}
```

Comment:
That could instead be the right interpreter from the start, instead of relying on the interpretation of said variable for the host. It would be clearer to debug.

Reply:
I removed this, was from a previous attempt that didn't work.